rotkehlchen/db/utils.py

```diff
@@ -430,6 +431,9 @@ def db_tuple_to_str(
             f'for {data[5]} closed at timestamp {data[3]}'
         )
 
+    if tuple_type == 'optimism_transaction':
```

Comment:
Do we need this extra information? If it's just for debugging and printing then just don't do it. KEep it all as evm transaction.